### DISCUSSION TITLE
Playlist History

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [Shabinder/SpotiFlyer#167] [Ideas]
What is not possible is that you can delete empty playlists and playlists that you no longer need.

![emptyhistory](https://user-images.githubusercontent.com/61406324/120905500-b3238700-c652-11eb-993c-4d1898cda26b.jpg)

**Comment by Shabinder** [comment]
How were you able to get a empty playlist there ?

I will maybe add History Delete Functionality in next update

**Reply by mondlicht-und-sterne** [reply]
> How were you able to get a empty playlist there ?

Good question, but it says so in my case.
